sokol_gfx.h

```diff
@@ -7037,6 +7044,8 @@ _SOKOL_PRIVATE void _sg_gl_reset_state_cache(void) {
 
 _SOKOL_PRIVATE void _sg_gl_setup_backend(const sg_desc* desc) {
     _SOKOL_UNUSED(desc);
+    SOKOL_ASSERT(desc->context.gl.default_framebuffer_cb == NULL || desc->context.gl.default_framebuffer_userdata_cb == NULL);
```

Comment:
You can omit the `== NULL`, or at least replace the `NULL` with `0` (yeah I know theoretically a NULL pointer doesn't have to be zero, but that's a very esoteric assumption).

(I would prefer removing the `== NULL` alltogether.

Reply:
removing `== NULL` changes the meaning. I think you're thinking of `!= NULL` being removable

Reply:
ah right... I was confused because in other places I have asserts like this:

`SOKOL_ASSERT(desc->context.d3d11.render_target_view_cb || desc->context.d3d11.render_target_view_userdata_cb);`

...but in this case, one of the two callbacks *must* be set. I wonder if the assert is necessary at all, because it's ok that both callbacks can 0, and if for some reason both callbacks are set, then one of the two will be called anyway (a proper check would be "either both callbacks are zero, or exactly one callback is provided").

Reply:
Replaced NULL with 0. I noticed there are 19 other places that NULL is used in sokol_gfx.h, how do you decide when to use NULL and when to use 0?